test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -2857,6 +2859,67 @@ public void PackageBuilderPreserveFileLastWriteTime()
             }
         }
 
+        [Theory]
+        [InlineData(true, true)]
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        [InlineData(false, false)]
+        public void EmitRequireLicenseAcceptance(bool emitRequireLicense, bool requireLicense)
```

Comment:
Although I suggested created this test with all cases in the same method, It would be better to separate succeeding and failing cases in separate methods.

See https://github.com/NuGet/NuGet.Client/pull/3716#pullrequestreview-525229595

Reply:
Done. I separated out a further category as well (ShouldNotEmitElement).